pkg/controller/operators/operatorcondition_controller.go

```diff
@@ -93,39 +93,34 @@ var _ reconcile.Reconciler = &OperatorConditionReconciler{}
 
 func (r *OperatorConditionReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	// Set up a convenient log object so we don't have to type request over and over again
-	log := r.log.WithValues("request", req)
-	log.V(2).Info("reconciling operatorcondition")
+	log := r.log.WithValues("request", req).V(1)
```

Comment:
`V(1)` is debug level for the underlying logr implementation, `zapr`.

Reply:
lol - of course it is. Why would you call it...`Debug`...?

Reply:
`logr` doesn't have named levels, really. It has error logs and info logs. info logs can have varying degrees of importance, so the number reflects the verbosity. Higher number = higher verbosity.

However logr is just an interface. There's an implementation of this interface for all the major logging libraries, so if the underlying implementation has named levels, they end up being mapped to a verbosity number in logr.

Reply:
Also... in logr, the verbosity is additive. So `log.V(1).V(1)` is the same as `log.V(2)`, It's somewhat misleading to say `log.V(1)` is equivalent to `Debug` since it depends on what the base verbosity of `log` is.

Reply:
Yeah, I'm not sure I'm a huge fan of the `logr` api from a user perspective.